js/kucoin.js

```diff
@@ -210,6 +211,28 @@ module.exports = class kucoin extends Exchange {
         return result;
     }
 
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        let currency = this.currency (code);
+        let response = await this.privateGetAccountCoinWalletAddress ({
+            'coin': currency['id'],
+        });
+        if ('success' in response) {
+            if (response['success'] && typeof response.data != 'undefined') {
```

Comment:
`response.data` is JS-only (not portable to Python)

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

> all associative keys must be single-quoted strings everywhere, `array['good']`, `array.bad`

Moreover, if there's no `'data'` key in response, we can't dereference it like we're used to do in JS, we have to check if the key is there first, like you do with the `'success'` key:

```
if ('data' in response) {
    if (typeof response['data'] !== 'undefined') {
    }
}
```

Reply:
Anyway, I'll edit it upon merging, no worries.